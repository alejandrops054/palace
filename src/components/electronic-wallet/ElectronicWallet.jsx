/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import TableProduct from './TableProduct'
import TableReference from './TableReference'
const ElectronicWallet = () => {
    return (
        <div className="card">
            <div className="card-body table">
                <TableProduct />
                <br/>
                <TableReference />
            </div>
        </div>
    );
}
export default ElectronicWallet;