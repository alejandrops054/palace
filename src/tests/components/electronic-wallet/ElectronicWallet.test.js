/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import ElectronicWallet from '../../../components/electronic-wallet/ElectronicWallet'

describe('ElectronicWallet component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<ElectronicWallet />)
        expect(wrapper).toMatchSnapshot();
    })

})