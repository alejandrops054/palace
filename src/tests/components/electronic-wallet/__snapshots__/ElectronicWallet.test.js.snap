// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ElectronicWallet component debe mostrar snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body table"
  >
    <TableProduct />
    <br />
    <TableReference />
  </div>
</div>
`;
